test/feature-benchmark/scenarios.py

```diff
@@ -1482,3 +1482,43 @@ def benchmark(self) -> BenchmarkingSequence:
 """
             ),
         ]
+
+
+class HydrateIndex(ScenarioBig):
+    """Measure the time it takes for an index to hydrate when a cluster comes online."""
+
+    def init(self) -> list[Action]:
+        return [
+            self.table_ten(),
+            TdAction(
+                """
+> CREATE CLUSTER idx_cluster SIZE '16', REPLICATION FACTOR 1
+> CREATE TABLE t1 (f1 INTEGER, f2 INTEGER)
+"""
+            ),
+        ]
+
+    def benchmark(self) -> MeasurementSource:
+        return Td(
+            f"""
+> SET CLUSTER = default
+> DROP INDEX IF EXISTS i1
+> DELETE FROM t1
+> ALTER CLUSTER idx_cluster SET (REPLICATION FACTOR 0)
+> CREATE INDEX i1 IN CLUSTER idx_cluster ON t1(f1)
+> INSERT INTO t1 (f1) SELECT {self.unique_values()} FROM {self.join()}
+> UPDATE t1 SET f1 = f1 + 100000
+> UPDATE t1 SET f1 = f1 + 1000000
+> UPDATE t1 SET f1 = f1 + 10000000
+> UPDATE t1 SET f1 = f1 + 100000000
+> UPDATE t1 SET f1 = f1 + 1000000000
+> SELECT 1
+  /* A */
+1
+> ALTER CLUSTER idx_cluster SET (REPLICATION FACTOR 1)
```

Comment:
This is a neat trick! One thing you can consider is using `? EXPLAIN` to make sure that the SELECT COUNT(*) query will use the index. I can totally envision a future optimization or a regression that will cause some other evaluation strategy to be chosen.

Reply:
I'll use max instead of count, that should always stay faster using the index.

Reply:
Ok, max was weird, used EXPLAIN instead.